### ISSUE TITLE
Docker image on https://hub.docker.com/ is 1 year old

### ISSUE THREAD

**Issue by oliv3** [benhutchins/docker-mediawiki#12]
Would it be possible to upload a recent image on hub.docker.com ?

**Comment by benhutchins** [comment]
Yes, just need to up the mediawiki version number it builds with to use the latest patches as well. Give me 24 hours, I'll update it this evening.
